lib/ohai/plugins/linux/platform.rb

```diff
@@ -113,6 +129,15 @@ def os_release_file_is_cisco?
       # no way to determine platform_version in a rolling release distribution
       # kernel release will be used - ex. 3.13
       platform_version `uname -r`.strip
+    elsif (os_release_info = os_release_file_is_cisco?)
+      # Cisco platform not based on known distro above
+      if os_release_info['ID'] == 'nexus'
+        platform 'nexus'
+        platform_family 'cisco-wrlinux'
+      elsif os_release_info['ID_LIKE'] =~ /cisco-wrlinux/
+        platform_family 'cisco-wrlinux'
+      end
+      platform_version os_release_info['VERSION'] || ""
```

Comment:
In case this key isn't in the file, seemed better to set platform_version to an empty string instead of nil. If that's an incorrect design, it can be removed.